grimoirelab_metrics/cli.py

```diff
@@ -172,6 +177,18 @@ def grimoirelab_metrics(
             else:
                 package_metrics["packages"][package] = {"metrics": None}
 
+        package_metrics["metadata"] = {
+            "started_at": start_date.isoformat(),
+            "finished_at": datetime.datetime.now(datetime.UTC).isoformat(),
+            "from_date": from_date.isoformat() if from_date else None,
+            "to_date": to_date.isoformat() if to_date else None,
+            "code_file_pattern": code_file_pattern if code_file_pattern else FILE_TYPE_CODE,
+            "binary_file_pattern": binary_file_pattern if binary_file_pattern else FILE_TYPE_BINARY,
+            "pony_threshold": pony_threshold,
+            "elephant_threshold": elephant_threshold,
+            "dev_categories_thresholds": dev_categories_thresholds,
```

Comment:
Can we add this to `["metadata"]["configuration"]`? So configuration parameters are organized and someone who doesn't know about the file can understand what they are.